components/ledger/internal/bootstrap/config.go

```
@@ -163,234 +283,412 @@ func InitServersWithOptions(opts *Options) (*Service, error) {
+		metricsFactory:        rmq.metricsFactory,
 	}, nil
 }
 
-type unifiedRouteSetup struct {
-	onboardingRouteOptions  *midazhttp.ProtectedRouteOptions
-	transactionRouteOptions *midazhttp.ProtectedRouteOptions
-	ledgerRouteOptions      *midazhttp.ProtectedRouteOptions
-	onboardingMongoManager  *tmmongo.Manager
-	transactionMongoManager *tmmongo.Manager
+// initLogger returns the injected logger from options, or creates a new one.
+func initLogger(opts *Options, cfg *Config) (libLog.Logger, error) {
```

golangci-lint (2.11.3)

[error] func initLogger is unused

(unused)